openedx/core/djangoapps/schedules/README.rst

```diff
@@ -365,9 +374,30 @@ emails in a variety of email clients to ensure there are no bugs in any
 of them. EdX uses a tool called `Litmus <http://litmus.com/>`__ that
 automates this process.
 
-To begin using Litmus, make sure that ACE is configured to use Sailthru,
-and then refer to the confluence page on `How to test emails in a
-variety of
-clients <https://openedx.atlassian.net/wiki/spaces/RET/pages/216563991/How+to+test+emails+in+a+variety+of+clients>`__
-which will explain how to send emails to Litmus using the
-``--override-recipient-email`` option to the management commands.
+To begin using Litmus, follow these steps:
+
+1. Make sure that ACE is configured to use Sailthru (see instructions above).
+2. Go to the `Litmus checklist page <https://litmus.com/checklist>`__ and start
+   a new checklist.
+3. The checklist will provide you with an email address to send the test email
```

Comment:
"at which you will send a test email."

Reply:
Isn't it "to which" not "at which"? I don't really know. It just sounds less awkward to me.

Reply:
Yup, you're right.